app/connectors/VatReturnConnector.scala

```diff
@@ -38,7 +38,7 @@ class VatReturnConnector @Inject()(config: Configuration, httpClient: HttpClient
     httpClient.POST[VatReturnRequest, VatReturnResponse](url, vatReturnRequest)
   }
 
-  def submitWithCorrection(vatReturnRequest: VatReturnWithCorrectionRequest)(implicit hc: HeaderCarrier): Future[VatReturnWithCorrectionResponse] = {
+  def submit(vatReturnRequest: VatReturnWithCorrectionRequest)(implicit hc: HeaderCarrier): Future[VatReturnWithCorrectionResponse] = {
```

Comment:
Is there a particular reason for renaming this?

Reply:
It was part of the refactoring of the CYA page - the intention was to overload the submit method, it looks like this wasn't needed in the end! Would you like it changed back?

Reply:
I think so, if the overloading isnt needed, lets have it more descriptive